src/puppetbridge.ts

```diff
@@ -878,7 +935,13 @@ export class PuppetBridge extends EventEmitter {
 		}
 		const puppetMxid = await this.provisioner.getMxid(room.puppetId);
 		if (event.sender !== puppetMxid) {
-			return; // this isn't our puppeted user, so let's not do anything
+			if (!this.config.relay.enabled) {
+				return; // relaying not enabled, don't allow message from other user
+			} else if (!this.provisioner.canRelay(event.sender)) {
+				return; // no permissions to be relayed
+			}
+			await this.applyRelayFormatting(event.room_id, event.sender, event.content);
+			event.sender = puppetMxid;
```

Comment:
what is this needed for?

Reply:
I wasn't sure if something checks the sender afterwards so I added that just in case